src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -59,15 +75,20 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handleOnAnnotator = (annotator: Annotator): void => {
+            this.setState({ annotator });
```

Comment:
If we're passing the annotator instance around, it seems to defeat the purpose of the other portions of this component. Can we avoid storing the annotator instance?